### ISSUE TITLE
Element:alias setAttributeNode to setAttributeNodeNS and setNamedIem to setNamedItemNS

### ISSUE THREAD

**Issue by jinhoward** [w3c/dom#70]
- [ ] https://github.com/whatwg/dom/commit/a0042ff  alias setAttributeNode to setAttributeNodeNS and setNamedIem to setNamedItemNS

**Comment by yongsheng** [comment]
Another issue addressing 'replace an attr' in Element:
- [ ] https://github.com/whatwg/dom/commit/f8089df Fix the set an attribute algorithm

**Comment by yongsheng** [comment]
depends on #96

**Comment by yongsheng** [comment]
wucongdonglai will take it.

**Comment by yongsheng** [comment]
These 2 issues were fixed by the patch when for #28,.
